### ISSUE TITLE
Bulma Calendar - maxDate throws an error

### ISSUE THREAD

**Issue by babajidemm-bd** [Wikiki/bulma-calendar#219]
## Bug Report

**Source**
https://cdn.jsdelivr.net/npm/bulma-extensions@6.2.7/bulma-calendar/dist/js/bulma-calendar.min.js

**Environment**
- Bulma version: v6.2.7
- OS: [e.g. OSX 10.13.4, Windows 10]
- Browser: Chrome XX (I have not tested other browsers but I guess this bug is not browser specific)
- How you are customizing the extension: no

**Current Behavior**
```js
Uncaught RangeError: Invalid array length
    at datePicker._renderDays (bulma-calendar.min.js:1)
    at datePicker.refresh (bulma-calendar.min.js:1)
    at datePicker.render (bulma-calendar.min.js:1)
    at bulmaCalendar._build (bulma-calendar.min.js:1)
    at bulmaCalendar._init (bulma-calendar.min.js:1)
    at new bulmaCalendar (bulma-calendar.min.js:1)
    at bulma-calendar.min.js:1
    at NodeList.forEach (<anonymous>)
    at Function.attach (bulma-calendar.min.js:1)
    ...
```

**Input Code**
- Initialization code:

```js
bulmaCalendar.attach('input[name="birth_date"][type=date]', {
            type: 'date',
            maxDate: new Date(),
        });
```

**Steps to reproduce** 
See input/initialization code above

**Comment by johannesE** [comment]
I get the exact same error, but only for my first date field on the page and only in chrome. I'm not setting maxDate at all. Using the workaround described in #211 does not change anything.

```
let lang = window.navigator.language || 'de';

  const calendars = bulmaCalendar.attach('[type="date"]', {
    lang: lang,
    dateFormat: 'YYYY-MM-DD',
    weekStart: 1
  });
```

**Comment by zafranf** [comment]
i fix this by using tomorrow and disable the tomorrow..

```
    let tomorrow = new Date();
    tomorrow.setTime(tomorrow.getTime() + 86400000);
    let calendars = bulmaCalendar.attach('[type="date"]', {
      maxDate: tomorrow,
      disabledDates: [tomorrow],
      ...
    })
```

**Comment by michael-hack** [comment]
Should be fixed with 6.1.5
